elastic4s-client-http4s/src/main/scala/com/sksamuel/elastic4s/http4s/Http4sClient.scala

```diff
@@ -24,18 +24,49 @@ object Http4sClient {
       override def run[A](fa: IO[A], cb: Either[Throwable, A] => Unit): Unit = fa.unsafeRunAsync(cb)
     }
 
+    Http4sClient(
+      client = client,
+      endpoint = endpoint,
+      runner = ioRunner
+    )
+  }
+
+  def usingIO(
```

Comment:
Maybe worth totally replace initial `usingIO` method? I don't think someone already using it, it haven't went into the release yet

Reply:
Actually I see, it went into `v8.15.4`. Still maybe worth replacing it...

Reply:
I think this makes sense, did remove the other `usingIO` function.